tests/helpers/modal-asserts.js

```diff
@@ -25,7 +25,7 @@ export default function registerAssertHelpers() {
     message = message || 'Dialog triggered by ' + options.openSelector + ' failed to open and close';
     var dialogContent = [dialogSelector, ':contains(' + options.dialogText + ')'].join('');
     var self = this;
-    return click(options.openSelector).then(function() {
+    return click(options.openSelector, options.context).then(function() {
```

Comment:
These `options.context` lines are a result of #58 to test the `dialogCloseSelector` while using tether.